frontend/app/src/components/home/thread/Thread.svelte

```diff
@@ -151,7 +153,16 @@
     }
 
     function fileSelected(ev: CustomEvent<AttachmentContent>) {
-        draftThreadMessages.setAttachment(threadRootMessageIndex, ev.detail);
+        const content = ev.detail;
```

Comment:
Could we deal with all this inside `Footer` and only bubble the event if the file can actually be attached? That way we don't have this duplicated. Footer already knows which "mode" it is running in.